packages/js/client/src/__tests__/core/plugin-subinvoke.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { Client, PluginModule } from "@polywrap/core-js";
+import { PolywrapClient } from "../..";
+
+describe("plugin-subinvoke", () => {
+  const mockPlugin = () => {
+    class MockPlugin extends PluginModule<Record<string, unknown>> {
+      public async call(input: {input: Uint8Array}, client: Client): Promise<ArrayBuffer> {
+        const res = await client.invoke({
+          uri: "ens/http.polywrap.eth",
+          method: "get",
+          input: input.input,
+          noDecode: true,
+        });
+        return res.data as ArrayBuffer;
+      }
+    }
+  
+    return {
+      factory: () => new MockPlugin({}),
+      manifest: {
+        schema: ``,
+        implements: [],
+      },
+    };
+  };
+
+  test("call", async () => {
+    const client = new PolywrapClient({
+      plugins: [
+        {
+          uri: "ens/demo.eth",
+          plugin: mockPlugin(),
+        },
+      ],
+    });
+
+    const result = await client.invoke({
+      uri: "ens/demo.eth",
+      method: "call",
+      input: {
+        input: Uint8Array.from([
```

Comment:
Would it be possible to have everything as string (or json) and then convert it to `Uint8Array`? This way we understand more what the input is